src/track/serato/tags.cpp

```diff
@@ -0,0 +1,62 @@
+#include "track/serato/tags.h"
+
+namespace mixxx {
+
+double SeratoTags::findTimingOffsetMillis(const QString& filePath) {
+    // TODO: Find timing offset using mp3guessenc
+    Q_UNUSED(filePath);
+
```

Comment:
@ywwg Here is the spot where we'd need to determine the timing offset.

The corresponding code for Rekordbox is here:
https://github.com/mixxxdj/mixxx/blob/master/src/library/rekordbox/rekordboxfeature.cpp#L934-L975